index.rst

```diff
@@ -1,54 +1,148 @@
+Furthermore, a couple runs where also dedicated to taking initial datasets for pipeline testing.
 
-  To add images, add the image file (png, svg or jpeg preferred) to the
-  _static/ directory. The reST syntax for adding the image is
+Over the course of analyzing the dataset taken over these runs, we realized that the WCS information in the header did not match the expected field orientation.
+Further attempts to refine the WCS with matching algorithms where unsuccessful, suggesting that the information in the header where wrong by a considerable amount.
+Analysis of longer exposures (300s) and sequence of exposures of the same field showed clear signs of field rotation.
+At the same time, analysis of mount telemetry showed no sign of induced jitter.
 
-  .. figure:: /_static/filename.ext
-     :name: fig-label
+After consulting with Observatory Sciences, the pointing component vendors, it was acknowledged that it was possible that the Nasmyth rotation angles could have been setup incorrectly.
+We then proceeded to investigate this possibility.
+Using the time when the images where taken and the length of the exposure we queried the EFD for the mount telemetry data and pointing information.
+The data was used to compute the expected sky position angle for a group of exposures (considering that the rotator was in the wrong direction) and the data was verified by eye first and later by catalog matching.
+We also used a long exposure, taken by accident during one of the earliest runs, to verify this hypothesis.
+We confirmed that the arcs formed by stars in the field are consistent with the expected arc-size if the rotator is moving in the wrong direction.
 
-     Caption text.
+Following is a description of the analysis performed in the data to demonstrate that the AT Nasmyth rotator is being commanded by the pointing component in the wrong direction.
+The pointing component will be updated to fix the issue.
 
-   Run: ``make html`` and ``open _build/html/index.html`` to preview your work.
-   See the README at https://github.com/lsst-sqre/lsst-technote-bootstrap or
-   this repo's README for more info.
+.. _section-data-analysis:
 
-   Feel free to delete this instructional comment.
+Data Analysis
+=============
 
-:tocdepth: 1
+The analysis focused on two different datasets; a group of exposures of well known easy-to-match targets and a long (>300s) exposure.
+:numref:`table-dataset` contains a summary of the data used for this analysis.
 
-.. Please do not modify tocdepth; will be fixed when a new Sphinx theme is shipped.
+.. _table-dataset:
 
-.. sectnum::
+.. table:: Journal of observations used for the rotator analysis.
 
-.. TODO: Delete the note below before merging new content to the master branch.
+    +------------+--------+------------+-----------+-----------+----------+----------+-----------+
```

Comment:
out of curiosity, was this table made by hand or did you use some sort of tool? Doing them by hand is a pain...